core/engine/src/main/kotlin/com/gatehill/corebot/config/Settings.kt

```diff
@@ -17,6 +17,7 @@ object Settings {
         val channelNames by lazy {
             (System.getenv("SLACK_CHANNELS") ?: "corebot").split(",").map(String::trim)
         }
+        val replyInThread by lazy { (System.getenv("REPLY_IN_THREAD")?.toBoolean() ?: false) }
```

Comment:
Please can this be moved to the settings object for consistency with other options